app/templates/index.html

```diff
@@ -5,34 +5,21 @@
 {% endblock %}
 {% block content %}
 
-<div class="grid-row main-wrapper--lg">
-	<div class="grid-column-full">
-		<h2>Tasks</h2>
-		<div class="page--content">
-			<p>Pick a task length.</p>
-			<div class="button-group" role="group" aria-label="Basic example">
-				<button type="button" class="button" onclick="handleClick('small')">50 participants</button>
-				<button type="button" class="button" onclick="handleClick('large')">500 participants</button>
-			</div>
-		</div>
-	</div>
+<div class="grid-row">
+  <div class="grid-column-two-thirds">
+    <div class="page--content">
+    <h2 class="heading-md">Before you start</h2>
+    <ol>
+      <li><strong>This tool will only work with data in specific formats.</strong> You must have two files – <code>mentees.csv</code> and <code>mentors.csv</code> – to start the matching process. Those files need to be formatted with specific column headings. You can <a href="#">download a template</a> from this website.</li>
+      <li><strong>Do not close your browser window during the matching process.</strong> Doing this may cause the matching process to fail and you to lose your matches.</li>
+      <li><strong>The larger the number of matches you need, the longer this takes.</strong> Matching mentors to mentees takes around 10 minutes to match 500 participants.</li>
+    </ol>
+    <hr>
+    <div class="button-group">
+      <a href="/match" class="button button--action">Start now</a>
```

Comment:
Shouldn't this go to the `upload` route? I don't see where the user goes next from `match`

Reply:
so maybe `"{{ url_for('main.upload' }}"`

Reply:
> Shouldn't this go to the `upload` route? I don't see where the user goes next from `match`

Yes, it should if the full journey is now in place. 

When I forked, the page was still the dummy version you'd produced.